.tree-sitter-lint/local_rules/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 use std::sync::Arc;
 
-use tree_sitter_lint::{FromFileRunContextInstanceProviderFactory, Rule};
```

Comment:
Was able to get the `better_any::Tid` (aka `Any` equivalent) stuff working so therefore could pass the instance provider factory/instance provider as trait objects so could get rid of these generics